### ISSUE TITLE
Missing Visual Focus on Carousel controls

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4296]
![Screen Shot 2019-08-28 at 10 23 11 PM](https://user-images.githubusercontent.com/15113219/63905595-b5b76e00-c9e2-11e9-9267-7c2e9a18d2c8.png)

Carousel **arrows** and **indicators** are both receiving focus but isn't visible.

UPDATE: **arrows** and **indicators** no longer become focuses, but we can't see the focus product after the focus product is beyond the width of the slide. Should turn to next slide on next product focus.

**Comment by Zeyber** [comment]
Closing as updated issue will be fixed in #4674

**Comment by KateChuen** [comment]
As mentioned, dupe of #4674. Moving this to Done.
